Newsweek-clone using bootstrap

Jmagero opened the pull request:
In this project, I worked on making a whole website using the Bootstrap Framework. I worked on the elements we had seen previously such as Grid and Flexbox, and I included Media Queries to be able to make the website look exactly the same as the original for different screen sizes.

General requirements:
Add descriptive README file to your repositories.
Use meaningful names for your Pull Requests and add a short summary of its content
Use descriptive commit messages
Do not commit files that are not related to the project. Instead, add them to .gitignore file.
Do not commit inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.

majovanilla commented on index.html:
```diff
@@ -0,0 +1,2199 @@
+    <nav class=" nav-bar navbar-expand-lg navbar-light navigation border-bottom">
+      <div class=" collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="d-flex flex-row align-items-center nav-list font-weight-bold text-body justify-content-between">
+          <li class="nav-item"><a class="border-right" href="#">U.S</a></li>
+          <li  class="nav-item"><a class="border-right" href="#">World</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Business</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Tech & Science</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Culture</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Newsgeek</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Sports</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Health</a></li>
+          <li class="nav-item"><a class="border-right" href="#">The Debate</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Vantage</a></li>
+          <li class="nav-item"><a  href="#">Weather</a></li>
+          <li  class="nav-item nav-icons noPadding text-body d-flex flex-row">
+            <input class=" form-control border-0 search_input w-100" type="search" placeholder="Search" aria-label="Search">
+              <i class="fa fa-search text-body search-icon " aria-hidden="true"></i>
+          </li>
```
Your nav elements should occupy 100% width of the navbar space. Please adjust them.
Yours:
![Screenshot from 2020-04-22 12-59-37](https://user-images.githubusercontent.com/51926937/80017036-8c7bae80-8499-11ea-8361-6d897edbf1c7.png)

Original:
![Screenshot from 2020-04-22 12-59-43](https://user-images.githubusercontent.com/51926937/80017047-91d8f900-8499-11ea-9b27-25a76b58b9cf.png)

majovanilla commented on index.html:
```diff
@@ -0,0 +1,2199 @@
+                  </a>
+                  <a href="#">
+                    <p class=" politics hero_stripe">Pandemic Heroes</p>
+                  </a>
+                </div>
+                <div class="hero-body">
+                  <a href="#">
+                    <h2 class="mag_link2 article-title py-2">
+                      COVID-19 Linked to Changes to the Brains of Some Patients
+                    </h2>
+                  </a>
+                  <p class="card-text card-text-hero pb-2">
+                    Have the pollsters learned from their mistakes in the 
+                    2016 presidential election? Maybe. Maybe not
+                  </p>
+                </div>
+              </article>
+              <article class="col-sm-12 col-md-4 col-lg-4 w-100 border-0 d-flex flex-column">
+                <div>
+                  <a href="#">
+                    <img class=" rounded-0 hero-img w-100" src="https://d.newsweek.com/en/full/1578426/zoe-monterola-founder-six-feet-supplies.jpg" alt="image">
+                  </a>
+                  <a href="#">
+                    <p class=" politics hero_stripe">Pandemic Heroes</p>
+                  </a>
+                </div>
+                <div class="hero-body">
+                  <a href="#">
+                    <h2 class="mag_link2 article-title py-2">
+                      COVID-19 Linked to Changes to the Brains of Some Patients
+                    </h2>
+                  </a>
+                  <p class="card-text card-text-hero pb-2">
+                    Have the pollsters learned from their mistakes in the 
+                    2016 presidential election? Maybe. Maybe not
+                  </p>
+                </div>
+              </article>  
+            </div>
+          </section>
```
You have this section duplicated on your page. On large screens,  you can see both.

If you want to show the sections in a different order depending on the screen size, this is not the way. There is an option called `order` (check it [here](https://css-tricks.com/almanac/properties/o/order/)) that you should use to arrange items (there's a class on bootstrap, too).

majovanilla commented on index.html:
```diff
@@ -0,0 +1,2199 @@
+        
+
+      </section>
+
+      <!-- first section on large screen -->
+
+      <div class="row first-section-wrapper">
```
You shouldn't do this either. The good practice of responsive pages is not creating duplicated content, but playing with classes to re-arrange, hide, and show the elements you want. Please delete the duplicated code and use bootstrap to arrange the elements like the original page.

majovanilla commented on index.html:
```diff
@@ -0,0 +1,2199 @@
+            <div class=" justify-content-end">
+              <img class="img-fluid more-stories_img" src="https://d.newsweek.com/en/full/1581093/coronavirus-america-recoveries-40000-people.jpg" alt="more stories">
+            </div>
+            <div class="d-flex flex-column ml-3">
+              <div>
+                <a href="#">
+                  <h4 class=" more-stories-title">
+                    Restauranteur Delivers Oranges to Hospital Workers Amid Coronavirus Crisis
+                  </h4>
+                </a>
+              </div>  
+              <div>
+                <p class=" more-stories-paragraph text-body">
+                  "We all have to help each other," Matthew Fermin, 
+                  owner of East Harlem restaurant Santiago's 
+                  Beer Garden, told Newsweek.
+                </p>
+              </div>
+            </div>
+          </article>
+          <section class="heros-pandemic">
+            <a  class="first-section-title w-100" href="#">
+              <h2 class="font-weight-bolder">HEROES OF THE PANDEMIC</h2>
+            </a>
+            <div class="row heros-articles">
```
These two sections have a different look than the original page: 
The first article should be on one line, yours is not.
The heroes of the pandemic have different font-weight and color.

![Screenshot from 2020-04-22 13-00-57](https://user-images.githubusercontent.com/51926937/80018293-6b1bc200-849b-11ea-9098-c214d8135940.png)

Original:
![Screenshot from 2020-04-22 13-01-07](https://user-images.githubusercontent.com/51926937/80018329-7a027480-849b-11ea-8556-4f05bd5caeb0.png)

majovanilla commented on index.html:
```diff
@@ -0,0 +1,2199 @@
+                <a href="#">
+                  <img class="rounded-0 w-100 " src="https://d.newsweek.com/en/full/1581097/trump-flag-pollsters-2016-2020.jpg"  alt="image">
+                </a>
+                <a href="#">
+                  <p class="politics">POLITICS</p>
+                </a>
```
The tag should be positioned on the bottom of the image. Yours have a little space.
![Screenshot from 2020-04-22 12-59-59](https://user-images.githubusercontent.com/51926937/80018440-a0281480-849b-11ea-855d-eeaaaec4afc7.png)

majovanilla commented on index.html:
```diff
@@ -0,0 +1,2199 @@
+        <section class="mag-wrapper">
+          <div class="d-flex align-items-center justify-content-center border-bottom font-roboto w-100 py-3 mag-heading w-100">
+            <img class="newsweek-icon" src="./new_icon.PNG" alt="newsweek-logo">
+            <h2 class=" mag-title px-2">IN THE MAGAZINE</h2>
+          </div>
+
```
The images of this section have a different size than the original page. Please fix them.
Yours:
![Screenshot from 2020-04-22 13-18-09](https://user-images.githubusercontent.com/51926937/80018577-cf3e8600-849b-11ea-9ba2-abb452207bde.png)

Original:
![Screenshot from 2020-04-22 13-18-02](https://user-images.githubusercontent.com/51926937/80018588-d5ccfd80-849b-11ea-85d9-c3d66f364334.png)

The "see all features" button is on top of the image when you make the screen smaller. That shouldn't happen.

majovanilla commented on index.html:
```diff
@@ -0,0 +1,2199 @@
+            <section class="in_the_magazine wrapper_1">
+              <div class="d-flex align-items-center justify-content-center border-bottom font-roboto w-100 py-3 mag-heading w-100">
+                <img class="newsweek-icon" src="./new_icon.PNG" alt="newsweek-logo">
+                <h2 class=" mag-title px-2">EDITOR'S PICK</h2>
+              </div>
+                <div class="row">
```
The articles in this section have different image sizes and titles. Please make them look like the original page.
Please edit the same on all the screen sizes.
Yours:
![Screenshot from 2020-04-22 13-19-25](https://user-images.githubusercontent.com/51926937/80018706-0f056d80-849c-11ea-838b-d55fd06e1547.png)

Original:
![Screenshot from 2020-04-22 13-19-19](https://user-images.githubusercontent.com/51926937/80018725-1462b800-849c-11ea-9ee8-376412da648c.png)

majovanilla commented on index.html:
```diff
@@ -0,0 +1,2199 @@
+      </div>
+
+      <div class="row">
+        <div class=" row d-flex align-items-center justify-content-start font-roboto signup-call py-3">
+          <h2 class="us px-2 mag_link font-weight-bold">U.S.</h2>
+        </div>
+        <div class="row">
```
This section's three side articles have a different size (see how the pass the two articles on the left). Please make them look like the original page.
Also, include the rest of the sections that are missing.
Yours:
![Screenshot from 2020-04-22 13-21-10](https://user-images.githubusercontent.com/51926937/80019038-8804c500-849c-11ea-99e2-00501cfe4d69.png)

Original:
![Screenshot from 2020-04-22 13-21-28](https://user-images.githubusercontent.com/51926937/80019046-8b984c00-849c-11ea-8b7e-b5cab9023077.png)

When you are in the tablet version, the articles on the side are overflowing the ones on the left. Please fix this:
![Screenshot from 2020-04-22 13-27-52](https://user-images.githubusercontent.com/51926937/80019451-2a24ad00-849d-11ea-8523-370973ae460e.png)

On the mobile version this section doesn't look like the original page:
The top two articles should have 100% width.
Yours:
![Screenshot from 2020-04-22 13-37-23](https://user-images.githubusercontent.com/51926937/80020380-8d630f00-849e-11ea-9321-60cfe741bfe1.png)

Original:
![Screenshot from 2020-04-22 13-37-13](https://user-images.githubusercontent.com/51926937/80020393-918f2c80-849e-11ea-9f53-2d5583beafc6.png)

majovanilla commented on index.html:
```diff
@@ -0,0 +1,2199 @@
+            <article class="w-100 sub-container">
+              <div class="d-flex">
+                <div>
+                  <img class="sub-img w-100 " src="https://d.newsweek.com/en/full/1579878/april-17-24-2020-cover.jpg" alt="subscribe-img">
+                </div>
+                <div class="d-flex flex-column align-items-start pt-5 sub-paragraph">
+                  <h2 class="mag_link2 py-2">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h2>
+                  <p class="py-2">All Access + Weekly Delivery.
+                    Choose A Membership That's Perfect for You!</p>
+                  <div class="py-2">
+                      <button type="button" class="subscribe_btn">SUBSCRIBE ></button>
+                  </div>
+                </div>
+              </div>
+            </article>
+          </section>
+        </div>
+        <div class="col-md-12 col-lg-4 signup-wrapper">
+          <div class=" col-lg-4 d-flex align-items-center justify-content-start font-roboto">
+            <h2 class="font-weight-bold sub-title">NEWSLETTER</h2>
+          </div>
+          <article class="w-100  sign-up-container">
+            <div class="d-flex">
+              <div>
+                <img class="sign-up-img w-100" src="https://d.newsweek.com/en/full/1579878/april-17-24-2020-cover.jpg" alt="subscribe-img">
+              </div>
+              <div class="d-flex flex-column align-items-start sub-paragraph">
+                <h2 class="mag_link2 py-2">The Starting 5</h2>
+                <p class="py-2">See why nearly a quarter of a million subscribers
+                   begin their day with the Starting 5.
+                </p>
+                  <div class="py-2">
+                    <button type="button" class="subscribe_btn">SIGN UP ></button>
+                  </div>                
+              </div>
+            </div>
+          </article>
+
+        </div>
+    
```
This part doesn't look like the original page, yet. Take a look at the Subscribe div: it has two images together.
The Newsletter div has a different image. The titles of both have different font-size...
Yours:
![Screenshot from 2020-04-22 13-25-19](https://user-images.githubusercontent.com/51926937/80019300-edf14c80-849c-11ea-9bfd-0d3a52153cf6.png)

Original:
![Screenshot from 2020-04-22 13-25-27](https://user-images.githubusercontent.com/51926937/80019312-f184d380-849c-11ea-90cf-2b2842cf4d74.png)


On mobile, the section behaves differently on your page. Please make it look like the original.
Yours:
![Screenshot from 2020-04-22 13-38-53](https://user-images.githubusercontent.com/51926937/80020490-c307f800-849e-11ea-9855-bcd6b7e29332.png)

Original:
![Screenshot from 2020-04-22 13-39-01](https://user-images.githubusercontent.com/51926937/80020502-c69b7f00-849e-11ea-9083-474efc6808e7.png)

majovanilla commented on index.html:
```diff
@@ -0,0 +1,2199 @@
+        </article>
+      </section>
+
+      <section class="featured-stories d-sm-block d-md-none d-lg-none border-bottom">
+        <a  class="first-section-title w-100" href="#">
+          <h2 class="font-weight-bolder">FEATURED STORIES</h2>
+        </a>
```
On the mobile version you are repeating this section before More stories. That shouldn't happen.

majovanilla commented on index.html:
```diff
@@ -0,0 +1,2199 @@
+      </div>
+
+      <section class="more-stories d-md-none d-lg-none mt-5">
+        <a  class="first-section-title w-100" href="#">
+          <h2 class="font-weight-bolder border-bottom">MORE STORIES</h2>
+        </a>
+        <article class="more-stories-article d-flex border-bottom">
```
On the mobile version, the articles on this section are less wider than the original page. Please make them look like the original page:
![Screenshot from 2020-04-22 13-35-58](https://user-images.githubusercontent.com/51926937/80020198-3d844800-849e-11ea-94dc-0fd72802137c.png)


Original:
![Screenshot from 2020-04-22 13-36-06](https://user-images.githubusercontent.com/51926937/80020208-4117cf00-849e-11ea-90eb-85d4f85b4703.png)

majovanilla commented on index.html:
```diff
@@ -0,0 +1,2199 @@
+            <small class="footer-copyright text-muted text-uppercase">© 2020 Newsweek</small>
+          </div>
+        </div>
+        <ul class="d-flex justify-content-around social-icons mb-0">
+          <li><a href="#"><i class="px-2 text-muted fab fa-facebook-square"></i></a></li>
+          <li><a href="#"><i class="px-2 text-muted fab fa-twitter"></i></a></li>
+          <li><a href="#"><i class="px-2 text-muted fab fa-instagram"></i></a></li>
+          <li><a href="#"><i class="px-2 text-muted fab fa-tumblr"></i></a></li>
+          <li><a href="#"><i class="px-2 text-muted fab fa-linkedin-in"></i></a></li>
+        </ul>
+      </div>
```
Give more space in between the lines of the footer elements, they look cramped.
Also, check the way the elements rearrange when you change screen sizes.
![Screenshot from 2020-04-22 13-41-24](https://user-images.githubusercontent.com/51926937/80020638-fcd8fe80-849e-11ea-97e1-e77580aa8b8b.png)

Original:
![Screenshot from 2020-04-22 13-41-17](https://user-images.githubusercontent.com/51926937/80020676-07939380-849f-11ea-8e5f-61a76fd6a4d9.png)

majovanilla commented on style.css:
```diff
@@ -0,0 +1,569 @@
+  background-color: hsl(200, 9%, 93%);
+}
+
+.top-story-article {
+  padding: 0 0 20px;
+}
+
+.heros-articles {
+  min-height: 515px;
+  height: 100%;
+}
+
+.hero-img {
+  width: 100%;
+}
+
+.hero-img-h {
+  max-height: 149px;
+  height: 100%;
+}
+
+.hero-body {
+  padding: 15px 0 20px;
+}
+
+.card-text-hero {
+  margin: 0 0 10px;
+}
+
+.opinion-article {
+  padding: 0 0 10px;
+}
+
+.opinion_heading {
+  margin-left: 3%;
+}
```
All these classes should be bootstrap classes. Please change them.

Jmagero commented:
Always remember to check your HTML using the validator. You have some errors and warnings. Please fix them.
![image](https://user-images.githubusercontent.com/52098394/80254297-99f48c80-8683-11ea-93c7-376ae7a9185e.png)

Always remember to activate your repo in Stickler. I cannot see your linters working!!

You are using some hover effects that are not the same as the original page, please make them look like the Newsweek site.-Fixed

Check the commented code below and fix it.-Fixed

Jmagero commented:
fixed overlap
